Subject: add AC_SUBST(LD) to configure.in
From: Marc Kleine-Budde <mkl@pengutronix.de>

$(LD) is used in Makefile.am, but when using libtool $(LD) sems not to
substituted automatically. This patch adds the missing AC_SUBST().

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>

---
 configure.in |    1 +
 1 file changed, 1 insertion(+)

Index: DirectFB-1.0.1/configure.in
===================================================================
--- DirectFB-1.0.1.orig/configure.in
+++ DirectFB-1.0.1/configure.in
@@ -82,6 +82,7 @@ AC_PROG_CC
 ifdef([AM_PROG_AS],[AM_PROG_AS],[])
 AM_PROG_LIBTOOL
 AM_SANITY_CHECK
+AC_SUBST(LD)
 AC_ISC_POSIX
 AC_PROG_INSTALL
 AC_PROG_MAKE_SET
